compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/viewinterop/InteropViewHolder.skiko.kt

```diff
@@ -136,6 +136,12 @@ internal open class InteropViewHolder(
         val coreModifier = Modifier
             .trackInteropPlacement(this)
             .onGloballyPositioned { layoutCoordinates ->
+
+                // TODO: Remove workaround for missing [onPlaced] after merging https://r.android.com/3245141
+                if (!container.contains(this)) {
```

Comment:
We can consider using bool flag for place/unplace because this check looks heavy for such operation.

Reply:
Initially I've done it like that, let me revert that approach